docs/src/content/docs/pt-pt/resources/community-content.mdx

```diff
@@ -0,0 +1,131 @@
+		title="Arquitetura Frontend Híbrida com Astro e Starlight"
+		description="Maxi Ferreira e Ben Holmes constroem um site de documentação com Starlight, TinaCMS, e um Playground de API interativo com autenticação."
+	/>
+	<LinkCard
+		href="https://www.olets.dev/posts/comparing-docs-site-builders-vuepress-vs-starlight/"
+		title="Comparando construtores de site de documentação: VuePress vs Starlight"
+		description="Como estes dois frameworks se comparam?"
+	/>
+</CardGrid>
+
+## Receitas e Guias
+
+Receitas são tipicamente guias passo a passo, curtos e focados, que levam o leitor a completar um exemplo funcional de uma tarefa específica.
+
+As receitas são também uma ótima maneira de adicionar novas funcionalidades ou comportamentos para seu projeto Starlight bastando seguir simples instruções passo a passo!
+
+Outros guias podem explicar conceitos relacionados a determinada área de conteúdo, como por exemplo usar imagens ou trabalhar com MDX.
+
+Explore o conteúdo produzido pela comunidade e mantido pelos utilizadores Starlight:
+
+<CardGrid>
+	<LinkCard
+		href="https://www.webpro.nl/scraps/versioned-docs-with-starlight-and-vercel"
+		title="Documentação versionada com Starlight e Vercel"
+		description="Um guia para implementar versões separadas da documentação para cada versão principal de um projeto"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-heading-links"
+		title="Adicione links em cabeçalhos Starlight"
+		description="Um guia sobre como usar um plugin rehype para criar links para seções específicas da sua documentação"
+	/>
+	<LinkCard
+		href="https://blog.otterlord.dev/posts/starlight-sponsors/"
+		title="Adicione patrocinadores para seu site Starlight"
+		description="Um guia sobre implementar um componente de patrocinadores personalizado na barra lateral da sua documentação"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-og-images"
+		title="Adicione imagens Open Graph a Starlight"
+		description="Um guia para gerar imagens sociais e as metatags correspondentes para as suas páginas"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-third-party-icon-sets"
+		title="Use ícones de terceiros em Starlight"
+		description="Um guia sobre como usar unplugin-icons para expandir a seleção de ícones disponíveis para Starlight"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-custom-html-head"
+		title="Edite o elemento HTML head de páginas Starlight"
+		description="Aprenda sobre como adicionar conteúdo como fontes, scripts e web analytics"
+	/>
+	<LinkCard
+		href="https://dev.to/mrrobot/publishing-documentation-with-astro-starlight-691"
+		title="Publicando documentação com Astro Starlight"
+		description="Começando com a documentação Starlight"
+	/>
+
+</CardGrid>
+
+## Conteúdo de Vídeo
+
+Descubra os vídeos e canais com conteúdo Starlight, incluíndo transmissões ao vivo e conteúdo educacional.
+
+import YouTubeGrid from '~/components/youtube-grid.astro';
+
+### Videos sobre Astro
```

Comment:
The intended meaning of “Astro videos” in the English is that these are kind of “official” videos, so not videos *about* Astro, but videos *by* Astro might be more accurate?